content-services/7.2/develop/repo-ext-points/content-transformers-renditions.md

```diff
@@ -285,6 +288,37 @@ Unlike pipelines, it must not be blank.
 * `transformOptions` - A list of references to options required by
 the pipeline transformer.
 
+### Adding pipelines and failover transforms to a T-Engine
+
+Since Content Services version 7.2 and ATS 1.5:
+
+So far we have talked about defining pipelines and failover transforms in the Repository or ATS router pipeline files. It is also possible to add them to a T-Engine's configuration, even when they reference a transformer provided by another T-Engine. It is only when all transform steps exist that the pipeline or failover transform becomes available. Warning messages will be issued if step transforms do not exist.
+
+Generally it is better to add them to T-Engines to avoid having to add an identical entry to both the Repository and ATS Router pipeline files.
+
+Since Alfresco Transform Core 3.0.0, config files with additional pipelines or overrides can be provided directly to a T-Engine image through additional config files (which may be resources on the classpath or external files) that are specified with Spring Boot properties such as `transform.config.file.<filename>` or environment variables like `TRANSFORM_CONFIG_FILE_<filename>`. In case of issues with changing configuration of transform-core-aio T-Engine such approach may be necessary.
```

Comment:
```suggestion
Since Alfresco Transform Core 3.0.0, config files with additional pipelines or overrides can be provided directly to a T-Engine image through additional config files (which may be resources on the classpath or external files) that are specified with Spring Boot properties such as `transform.config.file.<filename>` or environment variables like `TRANSFORM_CONFIG_FILE_<filename>`. You can use this approach if you have issues with changing the configuration of the transform-core-aio T-Engine.
```